### PR TITLE
Fix muc validation in graphql

### PR DESCRIPTION
**Pull request by arcusfelis**
This PR addresses #3752

Proposed changes include:
* Check the result of make_jid
* Require from argument.

### CODE REVIEW

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/142877) / small_tests / 31c3b95f2f585704eba4c2d8c21c909f22fbf171
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3799/142877/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3799/142877/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-10-07_08.57.40/lib.mongooseim.logs/run.2022-10-07_08.57.41/suite.log.html)

File: src/mod_muc_api.erl [hunk 1]
```diff
@@ -115,7 +114,9 @@ create_instant_room(MUCDomain, Name, OwnerJID, Nick) ->
                 Error ->
                     Error
             end;
-        false ->
+        {true, error} ->
+            {invalid_input, "Room name or domain are invalid. Check the format of the arguments."};
```
Comment (kamilwaz) [inline src/mod_muc_api.erl:118]: I would change the error message to "Room name or domain are invalid" to keep it consistent with other messages (no dot, one sentence).

**Review by kamilwaz — CHANGES_REQUESTED** [review]

File: src/mod_muc_api.erl (hunk 1 above)
Comment (arcusfelis) [reply src/mod_muc_api.erl]: ok

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/144370) / small_tests / e1e6433092c90fdc77ca6f0925b44469e7ab7ff3
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3799/144370/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3799/144370/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-10-13_08.20.33/lib.mongooseim.logs/run.2022-10-13_08.20.34/suite.log.html)

**Review by chrzaszcz — APPROVED** [review]
Looks good

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/145897) / small_tests / 02603b4b34c8cc9b1a377f940b8a0c7d042b5a46
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3799/145897/small_tests.24.3.4) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3799/145897/small_tests.24.3.4/small/ct_run.mongooseim@localhost.2022-10-21_09.57.25/lib.mongooseim.logs/run.2022-10-21_09.57.26/suite.log.html)

**Review by kamilwaz — APPROVED** [review]
There was one small typo (_are_ to _is_) that I fixed it by myself. The rest was :+1:

**Pull request merged**